oci/spec_opts.go

```diff
@@ -1426,24 +1421,6 @@ func WithDevShmSize(kb int64) SpecOpts {
 	}
 }
 
-// tryReadonlyMounts is used by the options which are trying to get user/group
-// information from container's rootfs. Since the option does read operation
-// only, this helper will append ReadOnly mount option to prevent linux kernel
-// from syncing whole filesystem in umount syscall.
```

Comment:
I think we should keep the comments here to know why we use readonly mount .... no just remove it

Reply:
- Opened https://github.com/containerd/containerd/pull/8397